articles/active-directory-b2c/openid-connect.md

```diff
@@ -31,31 +31,31 @@ Azure AD B2C extends the standard OpenID Connect protocol to do more than simple
 
 When your web application needs to authenticate the user and run a user flow, it can direct the user to the `/authorize` endpoint. The user takes action depending on the user flow.
 
-In this request, the client indicates the permissions that it needs to acquire from the user in the `scope` parameter, and specifies the user flow to run. To get a feel for how the request works, try pasting the request into a browser and running it. Replace `{tenant}` with the name of your tenant. Replace `90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6` with the app ID of the application you've previously registered in your tenant. Also change the policy name (`{policy}`) to the policy name that you have in your tenant, for example `b2c_1_sign_in`.
+In this request, the client indicates the permissions that it needs to acquire from the user in the `scope` parameter, and specifies the user flow to run. To get a feel for how the request works, try pasting the request into a browser and running it. Replace `{tenant}` with the name of your tenant. Replace `90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6` with the app ID of an application you've previously registered in your tenant or [register a web application](tutorial-register-applications.md#register-a-web-application). Replace `{application-id-uri}/{scope-name}` with the **Application ID URI** and **Scope** of an application that you've previously registered in your tenant. Go to the Azure AD B2C blade, select **App registrations**, select **All applications** and select an application. Under the **Manage** section, select **Expose an API**. If it's preferable to start fresh, [register a new web api](add-web-api-application.md#app-registrations) and [configure scopes](add-web-api-application.md#configure-scopes). Also change the policy name (`{policy}`) to the policy name that you have in your tenant, for example `b2c_1_sign_in`.
 
 ```http
 GET https://{tenant}.b2clogin.com/{tenant}.onmicrosoft.com/{policy}/oauth2/v2.0/authorize?
 client_id=90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6
-&response_type=code+id_token
-&redirect_uri=https%3A%2F%2Faadb2cplayground.azurewebsites.net%2F
+&response_type=code+token
```

Comment:
```suggestion
&response_type=code+id_token
```

Reply:
Ah!
I didn't know about "hybrid flow" though the the code+id_token response type is not new to me, but I haven't used it and so I made an incorrect assumption that there was no use for it in the example either.
I will also add that the id_token that is returned when using response_type=code+id_token results in an ID token that when pasted in jwt.ms it isn't decoded.
jwt.io does try, and I can see both aud and sub claims, but it also tells me that the token is incomplete because of an invalid signature, which explains why jwt.ms "discards" it (but the subsequent ID token, post code exchange, is OK).

This is only to try to explain why I didn't understand why it was included.
I just think this might be a bit confusing for other users that read and test the examples.
But I am guessing this is something that can be tackled in another PR or another article :-)

Now I have learned about the hybrid flow and use-cases for it and I understand why it's part of the sample request.